index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [],
+    this.length = 0
```

Comment:
Okay, this is technically correct but ONLY for this case (an empty array); ideally, your length should adapt to whatever number it might have, with `this.items.length` here.